examples/dev-mashup/connect.js

```diff
@@ -0,0 +1,53 @@
+import enigma from 'enigma.js';
+
+export default function connect() {
+  const loadSchema = () => fetch('https://unpkg.com/enigma.js/schemas/3.2.json').then((response) => response.json());
+
+  const createConnection = () =>
+    loadSchema().then((schema) =>
+      enigma
+        .create({
+          schema,
+          url: `ws://${window.location.hostname || 'localhost'}:9076/app/${Date.now()}`,
```

Comment:
I'd like for us to be using QSE SaaS in these examples. I know in the past we've targeted Windows and Qlik Core, but we want to subtly infer the use of Qlik Sense Enterprise SaaS.

Reply:
sure, but this example is for specific development in Nebula. So will worry later about that.